src/pages/understand-stacks/command-line-interface.md

```diff
@@ -0,0 +1,126 @@
+
+```bash
+stx <command> -H "http://localhost:20443"
+```
+
+## Account
+
+This section describes how to use the CLI to manage an account.
+
+~> It is not recommended to use the CLI to handle accounts with real STX tokens on the mainnet. Using an appropriate wallet build to support secure token holding is recommended.
+
+### Creating an account
+
+You can generate a new account for testnet by using the `make_keychain` command with the `-t` option:
+
+```bash
+stx make_keychain -t
+```
+
+Your response should look like this:
+
+```json
+{
+  "mnemonic": "private unhappy random runway boil scissors remove harvest fatigue inherit inquiry still before mountain pet tail mad accuse second milk client rebuild salt chase",
+  "keyInfo": {
+    "privateKey": "381314da39a45f43f45ffd33b5d8767d1a38db0da71fea50ed9508e048765cf301",
+    "address": "ST1BG7MHW2R524WMF7X8PGG3V45ZN040EB9EW0GQJ",
+    "btcAddress": "n4X37UmRZYk9HawtS1w4xRtqJWhByxiz3c",
+    "index": 0
+  }
+}
+```
+
+The mnemonic is your 24 word seed phrase which you should back up securely if you want access to this account again in the future. Once lost, it cannot be recovered.
+
+The Stacks address associated with the newly generated account is:
+`ST1BG7MHW2R524WMF7X8PGG3V45ZN040EB9EW0GQJ`
+
+-> This is a testnet address for use with the testnet only
+
+It is best to store the response of the CLI somewhere. You will need the private key, for instance, to send tokens to others.
+
+### Checking balance
+
+You can check the balance of your account using the following command:
+
+```bash
+stx balance ST1BG7MHW2R524WMF7X8PGG3V45ZN040EB9EW0GQJ -t
+```
+
+The response will look like this:
+
+```json
+{
+  "balance": "10000",
+  "nonce": 0
+}
+```
+
+-> To receive testnet STX tokens, please use the faucet [faucet](https://testnet.blockstack.org/faucet)
+
+Take note that the nonce for the account is `0`. This number is important for transaction broadcasting.
+
+## Transactions
+
+This section describes how to use the CLI to generate and broadcast transactions.
+
+### Sending Tokens
+
+In order to send tokens, the CLI command requires 5 parameters:
+
+- **Recipient Address**: The Stacks address of the recipient
+- **Amount**: The number of Stacks to send denoted in microstacks (1 STX = 1000000 microstacks)
+- **Fee Rate**: The transaction fee rate for this transaction. You can safely set a fee rate of 1 for Testnet
+- **Nonce**: The nonce is a number that needs to be incremented monotonically for each transaction from the account. This ensures transactions are not duplicated
+- **Private Key**: This is the private key corresponding to your account that was generated when
+
+The CLI command to use with these parameters is `send_tokens`:
+
+```bash
+$ stx send_tokens ST2KMMVJAB00W5Z6XWTFPH6B13JE9RJ2DCSHYX0S7 1000 200 0 381314da39a45f43f45ffd33b5d8767d1a38db0da71fea50ed9508e048765cf301 -t
+
+d32de0d66b4a07e0d7eeca320c37a10111c8c703315e79e17df76de6950c622c
```

Comment:
We now respond with the txid and an explorer link

Reply:
good point, didn't look closely enough at this content

Reply:
I realize the references arent' updated either: https://docs.blockstack.org/references/blockstack-cli

Reply:
ping